.gitlab-ci.yml

```diff
@@ -441,7 +457,7 @@ init_workspace:
 build_client:
   only:
     variables:
-      - $BUILD_QEMUX86_64_UEFI_GRUB == "true"
+      - $BUILD_CLIENT == "true"
```

Comment:
I think we can simplify this further and remove `BUILD_CLIENT` and `BUILD_SERVERS`, using only `RUN_INTEGRATION_TESTS` as an alias for this use case. My reasoning is that there is no use case where a user would like to run integ tests but not build client or server. Note that as it is now it will just "work", because we are using OR operation everywhere with `RUN_INTEGRATION_TESTS`.

Does it make sense? If so I suggest then to remove it from here and the other jobs, and just use `RUN_INTEGRATION_TESTS` everywhere.

Reply:
There is one use case, and that is when building releases where you have already tested the components. You want to get artifacts that you can publish, but you don't want to run the whole suite. In that case these options are useful.

Reply:
:+1: